### PR TITLE
updated guzzle

### LINKED ISSUE #3: Update Guzzle dependency
Cool package! Thanks for creating it! I tried to install it in my Laravel project. Unfortunately, the package is not compatible with the latest Guzzle version which Laravel uses.

``` - Root composer.json requires laravel-notification-channels/expo dev-main -> satisfiable by laravel-notification-channels/expo[dev-main].

- laravel-notification-channels/expo dev-main requires guzzlehttp/guzzle ~6.0 -> found guzzlehttp/guzzle[6.0.0, ..., 6.5.x-dev] but it conflicts with your root composer.json require (^7.3.1).
```
Might be good to take care of this before the official release of your package.

Keep up the good work! 

Edit: I'll do some testing with v7 and create a PR for it.Then you can decide for yourself if you do something with it.

### PR DESCRIPTION
**Pull request by mttlck**
Hey,
this is just a simple update of guzzle, so we could use the channel in Laravel 8. 

Closes #3

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "require": {
         "php": ">=7.4 | ^8.0",
         "ext-json": "*",
-        "guzzlehttp/guzzle": "~6.0",
+        "guzzlehttp/guzzle": "~7.0",
```
Comment (atymic) [inline composer.json]: Can you allow both 6 and 7 please?

**Review by atymic — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (mttlck) [reply composer.json]: Of course, completely overlooked the bc break.

**Comment by chieft3ch** [comment]
This is also an issue for us! If this could get approved & merged would help a few people out. Thank you

**Comment by nicko170** [comment]
This is now redundant - merged in #5 - thanks!

**Pull request closed without merging**